package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "canonical-path": "0.0.2",
     "catharsis": "^0.8.1",
-    "change-case": "^2.1.0",
+    "change-case": "3.0.0",
```

Comment:
Alternatively we don't break at all by mapping the `sentenceCase` filter to the `noCase` function...